docs/source/basic/example.rst

```diff
@@ -0,0 +1,76 @@
+Code Example
+============
+
+The following example shows a small hello word example written with ``alpaka`` that can be run on different processors.
+
+.. literalinclude:: ../../../example/helloWorld/src/helloWorld.cpp
+   :language: C++
+   :caption: helloWorld.cpp
+
+Use alpaka in your project
+++++++++++++++++++++++++++
+
+``alpaka`` supports ``CMake`` to integrate it into your own project. There are two possible methods.
+
+Use alpaka via ``find_package``
+-------------------------------
+
+The ``find_package`` method requires ``alpaka`` to be :doc:`installed </basic/install>` in a location where ``CMake`` can find it. The following example shows a minimal example of a ``CMakeLists.txt`` that uses ``alpaka``:
```

Comment:
Maybe place the hint from below here, explaining what "where CMake can find it" means exactly.

Reply:
Do you mean the hint box from the `install.rst`?

Reply:
No, the one from this file (line 52). Or maybe just a link to the CMake 3.18 documentation explaining its search paths.